src/api/pricer/BaseExchangeApi.ts

```diff
@@ -1,6 +1,6 @@
 import type { BigNumber as BigNumberJs } from 'bignumber.js';
 
-export type BaseCurrency = 'TRIF' | 'RIF' | 'RDOC' | 'RBTC' | 'TKN';
+export type BaseCurrency = 'TRIF' | 'RIF' | 'RDOC' | 'RBTC' | 'TKN' | 'USDRIF';
```

Comment:
What is going to be the name in testnet, `USDRIF` aswell?

Reply:
Yes, it looks like that indeed. I've checked the name of the [testnet contract](https://explorer.testnet.rsk.co/address/0x8dbf326e12a9ff37ed6ddf75ada548c2640a6482).